kubectl-minio/cmd/cmd-operator-create.go

```diff
@@ -0,0 +1,219 @@
+/*
+ * This file is part of MinIO Operator
+ * Copyright (C) 2020, MinIO, Inc.
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/markbates/pkger"
```

Comment:
https://github.com/markbates/pkger/issues/58 known performance limitations in pkger, is there a better way to do the same? 

Probably a better library is https://github.com/rakyll/statik

Reply:
moved to statik